source/beamline/BioXAS/BioXASSideBeamline.cpp

```diff
@@ -332,50 +509,66 @@ void BioXASSideBeamline::setupComponents()
 
 	scaler_->setTriggerSource(zebraTriggerSource_);
 
-	// Scaler channel detectors.
-
-	i0Detector_ = new CLSBasicScalerChannelDetector("I0Detector", "I0 Detector", scaler_, 16, this);
-	connect( i0Detector_, SIGNAL(connected(bool)), this, SLOT(updateConnected()) );
-
-	i1Detector_ = new CLSBasicScalerChannelDetector("I1Detector", "I1 Detector", scaler_, 17, this);
-	connect( i1Detector_, SIGNAL(connected(bool)), this, SLOT(updateConnected()) );
-
-	i2Detector_ = new CLSBasicScalerChannelDetector("I2Detector", "I2 Detector", scaler_, 18, this);
-	connect( i2Detector_, SIGNAL(connected(bool)), this, SLOT(updateConnected()) );
-
-	// I0 channel amplifier.
+	// I0 channel.
 
-	i0Keithley_ = new CLSKeithley428("I0 Channel", "AMP1607-601", this);
+	i0Keithley_ = new CLSKeithley428("AMP1607-601", "AMP1607-601", this);
```

Comment:
Just a question about the name change, do you think it makes more sense to have the AM name be the PV base name rather than the contextual name (like `I0 Channel`)?